adoc/chapters/programming_interface.adoc

```diff
@@ -9956,16 +9958,32 @@ The characteristics of USM allocations are summarized in
 |                                 Another [code]#device#    | Optional  | Another [code]#device#| Optional
 |====
 
-USM allocations are only guaranteed to be valid in the SYCL context in which
-they were created. The behavior of accessing a USM allocation in a different
-context from which it was created is undefined.  Attempting to access a kind
-of USM allocation that is not supported by the device results in undefined
-behavior.
+Each USM allocation has an associated SYCL context, and any access to that
```

Comment:
Good point, I made this change for context.

I think there's some balance we need to find, though, because not every occurrence of the word "context" or "device" should be replaced with a link, otherwise, the text becomes hard to read.  My thinking is that we should use the link only on the first occurrence when describing some idea, and then only use the link if the glossary term is relevant to that idea.    Obviously, this is not a very precise rule, so there are a lot of grey areas.